### ISSUE TITLE
Accessibility: Links must have decernible text

### ISSUE THREAD

**Issue by bedmison** [web-cat/web-cat-subsystem-Core#7]
Ensure links have discernible text. To solve this problem, you need to fix at least (1) of the following:

- Element does not have text that is visible to screen readers
- aria-label attribute does not exist or is empty
- aria-labelledby attribute does not exist, references elements that do not exist or references elements that are empty
- Element has no title attribute

This issue shows up in the links that are being the table head icons that control the sorting of these columns:
- Done
- Name
- Time 
- Score

The urls are A tags that have JS void(0) in the href property, and embedded onClick() that points to a WO encoded url. IE:

`
<a href="javascript:void(0);" onclick="dojo.animateProperty({'duration':250,'node':'_0_0_0_29_1_6_1_0_2_1_1_1_0_0_1_1_1_0_3_0_6_1_submissionsTable_31843__tbody','properties':{'opacity':0.25}}).play();webcat.remoteSubmit(this, {handleAs:'javascript', form:dojo.query('form[name=f_0_0_0_29_1_9_0_7_0_1_1_1_0_0_1_1_11]')[0], sender:'0.0.0.29.1.9.0.7.0.1.1.1.0.0.1.1.11.1.0.0.1.1.5.1.0.1.4.1', url:'/Web-CAT/WebObjects/Web-CAT.woa/ajax/2.0.0.0.29.1.9.0.7.0.1.1.1.0.0.1.1.11.1.0.0.1.1.5.1.0.1.4.1'});">
`
